assets/sass/v2/_identify.scss

```diff
@@ -0,0 +1,75 @@
+.sign-in-with-device-option {
+  .okta-verify-container {
+    .link-button {
```

Comment:
why one-off the link-button? it seems odyssey design and seems not compatible with other elements?

Reply:
Same question. This is the only button that will be as per odyssey design ?

Reply:
This is the only button in SIW v2 with Odyssey design as far as I can tell. @nbories-okta ^